python docs update. (#95842)
Azure/azure-content: articles/azure-monitor/app/opencensus-python-request.md

    }
    ```

> You can find an example Django application in the sample Azure Monitor OpenCenses Python git repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).

Reviewer: ```suggestion
You can find an example Django application in the sample Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).
```

> You can find an example Django application in the sample Azure Monitor OpenCenses Python git repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).

Reviewer: I think it would be good to add all these messages at the END of the respective sections. Maybe word it like `You can fine ADDITIONAL django applications in the sample Azure...`

Author: Moved.

> 

Reviewer: Might want to link to [this](https://github.com/Azure-Samples/azure-monitor-opencensus-python/blob/master/docs_samples/02_python_app_insights_dependencies.md#configuring-django-applications) instead of just the root folder.

> You can find an example Django application in the sample Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor).

Reviewer: Please update these links to point to the READMEs of the respective samples once those get merged in the samples repo.

Author: Done.

> You can find an example Django application in the sample Azure Monitor OpenCensus Python samples repository located [here](https://github.com/givenscj/azure-monitor-opencensus-python/tree/master/azure_monitor/django_sample).

Reviewer: ```suggestion
You can find a Django sample application in the sample Azure Monitor OpenCensus Python samples repository located [here](https://github.com/givenscj/azure-monitor-opencensus-python/tree/master/azure_monitor/django_sample).
```

> ## Tracking Flask applications

Reviewer: Link to flask sample?

    
    > [!NOTE]
    > To run Flask under uWSGI in a Docker environment, you must first add `lazy-apps = true` to the uWSGI configuration file (uwsgi.ini). For more information, see the [issue description](https://github.com/census-instrumentation/opencensus-python/issues/660). 

>     You can find a Flask sample application that uses dependencies in the Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor/flask_sample).

Reviewer: ```suggestion
    You can find a Flask sample application that tracks requests in the Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor/flask_sample).
```
